chainmanager/client/rest/tx.go

```diff
@@ -0,0 +1,12 @@
+package rest
+
+import (
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/gorilla/mux"
+)
+
+// RegisterRoutes - Central function to define routes that get registered by the main application
+// RegisterRoutes registers the auth module REST routes.
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/chainmanager/params", paramsHandlerFn(cliCtx)).Methods("GET")
```

Comment:
All GET methods should be in query.go. 
As a good practice create rest.go for registering routes of tx.go and query.go.